.gitignore

```diff
@@ -2,6 +2,7 @@
 *.fits
 .ignore/
 docs/source/_build/
+docs/*env*/
```

Comment:
This would only ignore _folders_ named `*env*`; it does not match a `.env` file. Further, `.env` is already included below

```suggestion
```